lib/file_transfer_component/commands/initiate.rb

```diff
@@ -16,7 +16,16 @@ def self.build(name, uri, reply_stream_name: nil)
       def self.call(name, uri, reply_stream_name: nil)
         instance = build(name, uri, reply_stream_name: reply_stream_name)
         instance.()
-        instance.file_id
+      end
+
+      def file_id
+        @file_id ||= Identifier::UUID::Random.get
```

Comment:
`Identifier::UUID::Random` should be a `dependency`.

This line would become:
`@file_id ||= identifier.get`